deploying/serving-apps/requirements.txt

```
@@ -0,0 +1,23 @@
+annotated-types==0.6.0
+anyio==4.3.0
+click==8.1.7
+exceptiongroup==1.2.1
+fastapi==0.110.2
+h11==0.14.0
+idna==3.7
+joblib==1.4.0
+numpy==1.26.4
+pandas==2.2.2
+pydantic==2.7.0
+pydantic_core==2.18.1
+python-dateutil==2.9.0.post0
+pytz==2024.1
+scikit-learn==1.4.2
```

This is to avoid the warning when running inference.
```suggestion
scikit-learn==1.3.2
```